bundles/org.openhab.binding.wizlighting/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:wiz:light">
+		<parameter name="bulbMacAddress" type="text" required="true">
+			<label>Bulb MAC address</label>
+			<description>Mac Address of the bulb</description>
+			<required>true</required>
+		</parameter>
+		<parameter name="bulbIpAddress" type="text" required="true">
+			<label>Bulb IP Address</label>
+			<context>network-address</context>
+			<description>IP address of the bulb</description>
+			<required>true</required>
+		</parameter>
+		<parameter name="updateInterval" type="integer" min="5">
+			<label>Update interval</label>
+			<description>Update time interval in seconds to request the status of the bulb.</description>
+			<default>60</default>
+			<advanced>true</advanced>
+		</parameter>
+		<parameter name="useHeartBeats" type="boolean" required="false">
+			<label>Use heartbeats to synchronize with bulb</label>
```

Comment:
sounds more like a description than a label.
Also, the first letter of every word in a label should be capitalized.